cypress/integration/e2e/header-control-menu/return-to-now-button.spec.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+describe("'Come back to now' button", () => {
+  context("when schedule is loaded", () => {
```

Comment:
Ogólnie to przycisk wróć do teraz zadziała również bez wczytanego grafiku, więc wczytywanie nie jest konieczne.
Ale myślę, że skoro już idziemy w tę stronę to przydałoby się zdefiniować również test na to jak to działa `when schedule is not loaded`

Reply:
XD, myślałem że piszemy po angielsku review xDDD (mimo tego że czytałem polskie komentarze...), ciężkie poranki

Reply:
Zdarza się xd 

A generalnie, to piszemy po polsku, bo wtedy review zajmują mniej czasu (przynajmniej u mnie)